Syntax.in

```diff
@@ -0,0 +1,31 @@
+Syntax: 
+
+program = "{" compound_stm "}"
+
+compound_stm = stm[stm]
+
+declaration_stm = type IDENTIFIER ["=" expression]
+
+type = prim_type | arraydecl
+
+prim_type = "Integer" | "Boolean" | "Character" | "String" | "Float"
+
+arraydecl = "Array""<" type ">"
+
+assignment_stm = IDENTIFIER "=" expression
+
+read_stm = "read" IDENTIFIER ["," IDENTIFIER]
```

Comment:
Maybe { "," IDENTIFIER }, so that you could read more than two variables at most?